sleepecg/io/sleep_readers.py

```diff
@@ -15,15 +15,11 @@
 
 from ..config import get_config
 from ..heartbeats import detect_heartbeats
-from .nsrr import get_nsrr_url, list_nsrr
-from .utils import download_file
+from .nsrr import _get_nsrr_url, _list_nsrr
+from .utils import _download_file
 
-__all__ = [
-    'read_mesa',
-]
 
-
-class SleepStage(IntEnum):
+class _SleepStage(IntEnum):
```

Comment:
I'd actually be OK with leaving this as `SleepStage`. We explicitly import objects in `__init__.py` anyway, so we don't always have to be strict with our naming. Same for some constants that look weird with an underscore. Don't users at least see `SleepStage` objects sometimes because they get returned? But I'll leave it up to you.

Reply:
Users don't see it, since `SleepRecord.sleep_stages` is an integer array, but it would make sense to have it in the documentation, thanks for noticing!

I'd prefer to keep underscore prefixes for constants though.